### PR TITLE
Create uniform json schema for all sources

### PR DESCRIPTION
**Pull request by lego963**
Resolves: #32

### CODE REVIEW

File: delatore/sources/influx.py [hunk 1]
```diff
@@ -171,22 +165,15 @@ def influx_client(self):
...
-    def convert(cls, data: dict) -> str:
-        data.pop('From', None)
-        text = convert(data)
-        text = '*From InfluxDB*\n' + replace_emoji(text, INFLUX_EMOJI_MAP)
-        return text.strip()
+    def convert(cls, data: dict) -> dict:
+        return generate_message(cls.CONFIG_ID, data)
 
-    async def get_update(self) -> Optional[Json]:
+    async def get_update(self) -> List[Dict]:
```
Comment (outcatcher) [inline delatore/sources/influx.py]: Actual return value is dict here

File: delatore/sources/influx.py [hunk 2]
```diff
@@ -195,11 +182,14 @@ def convert(cls, data: dict) -> str:
             last_time = last_record.raw['series'][0]['values'][0][0]
             last_time_ms = _convert_time(last_time)
             now = datetime.utcnow().timestamp()
+
+            # TODO: if entity == LB DOWN = 30
+
             if now - last_time_ms < 300:
-                return 'OK'
-            return 'FAIL'
+                return generate_status(entity, 'OK')
```
Comment (outcatcher) [inline delatore/sources/influx.py]: Second argument should be of `Status` type

File: delatore/sources/http.py [hunk 3]
```diff
@@ -61,11 +65,17 @@ class AWXWebHookSource(HttpListenerSource):
...
-    def convert(cls, data: dict) -> str:
-        data.pop('From', None)
-        text = convert(data).strip()
-        text = '* From Ansible Tower *\n' + replace_emoji(text, AWX_LISTENER_EMOJI_MAP)
-        return text
+    def convert(cls, data: dict) -> dict:
+        try:
+            status = generate_status(
+                name=data['Name'],
+                status=data['Status'],
```
Comment (outcatcher) [inline delatore/sources/http.py]: Status should be converted to `Status` type

File: delatore/configuration/resources/sources.yaml [hunk 4]
```diff
@@ -9,6 +9,7 @@
       LB_DOWN: lb_down_test
       SCSI_HDD_TEST: iscsi_connection
       RDS_TEST: ce_result
+      AUTOSCALING: as_result
```
Comment (outcatcher) [inline delatore/configuration/resources/sources.yaml]: Please implement as another PR, it does not depend on other changes

**Comment by lego963** [comment]
TODO: if entity == LB DOWN = 30
AUTOSCALING AS as_result

**Comment by lego963** [comment]
LB_DOWN 15 min
AUTOSCALING 30 min

**Pull request merged**